Doc/library/ssl.rst

```diff
@@ -65,6 +65,7 @@ by SSL sockets created through the :meth:`SSLContext.wrap_socket` method.
 
    Use of deprecated constants and functions result in deprecation warnings.
 
+.. include:: ../includes/wasm-notavail.rst
```

Comment:
Typically ``availability`` is listed above ``versionadded``/``versionchanged``, at least in the various existing examples I checked. It would be good to try to be consistent about where users can expect to find this information, especially in Reference-focused docs such as this.

Reply:
I moved it below the notes. It seems more logical to me.